confidence.py

```diff
@@ -232,7 +237,7 @@ def get(self, path, default=_NoDefault, as_type=None, resolve_references=True):
             if as_type:
                 return as_type(value)
             elif isinstance(value, Mapping):
-                namespace = Configuration()
+                namespace = type(self)(separator=self._separator, missing=self._missing)
```

Comment:
The reverse for this is not implemented and can lead to rather tricky bugs. Creating a new `Configuration` based on an existing one will always use the default kwarg unless explicitly copied from the argument.

Reply:
Created #39.